python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -473,8 +482,10 @@ def run_through_rpc(measure_input, build_result,
         remote.upload(build_result.filename)
         func = remote.load_module(os.path.split(build_result.filename)[1])
         ctx = remote.context(str(measure_input.target), 0)
+        f_prepare = 'cache_flush_cpu_non_first_arg' if enable_cpu_cache_flush else ''
```

Comment:
The processing flow of this function seems a bit long:
1. setting `enable_cpu_cache_flush` to True
2. setting `f_prepare` to "cache_flush_cpu_non_first_arg"
3. setting `pf_prepare` to the packed function

To me, it's hard for people to realize "cache_flush_cpu_non_first_arg" is the global function symbol unless tracing back to `rpc_module.cc`. Is that possible to get the packed function here directly so that we can set the type of `f_prepare` to `PackedFunc`? It's also more general for its description (the prepared function) IMHO.

Reply:
Previous design I also want to pass the PackedFunc here directly but it will have problem in RPC mentioned here: https://github.com/apache/incubator-tvm/pull/5914#issuecomment-654560411 So currently we could only pass the string to work around rpc restriction.

Reply:
Ah sorry for missing the previous comment. Then we could leave it for now. Meanwhile, do you guys think it would be better to add a comment saying about this restriction?

Reply:
I could add it in the code place in the measure_methods.py. Do you accept this place?

Reply:
I'm fine with it. Thanks